config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/customers/:id", to: "customers#show"
+  patch "/customers/:id", to: "customers#update"
```

Comment:
As mentioned in the controller, a more RESTful option would be
`post "/customers/:customer_id/items", to: "customer_items#create"`

Reply:
It's nice for the path to include the 2 resources you're creating a relationship between - customer and items.